### DISCUSSION TITLE
Netdata Cloud

### DISCUSSION THREAD

**Discussion by olmedocr** [arjunkomath/netdata-ios#30] [Q&A] [answered]
Is there any reason this does not work with cloud? I just landed in netdata, so genuinely asking without malice.

**Comment by arjunkomath** [comment, marked as answer]
> Is there any reason this does not work with cloud? I just landed in netdata, so genuinely asking without malice. 

Netdata cloud is proprietary and is not open source, as far as I know there are no APIs to consume Netdata cloud.
